testing/test.gni

```diff
@@ -0,0 +1,413 @@
+# Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
This file is copied from Chromium, so I left the Chromium copyright and added the license text below.

Reply:
Thanks for calling that out.  Seems ok to me.